webapp-src/src/Login/SelectAccount.js

```diff
@@ -29,17 +29,19 @@ class SelectAccount extends Component {
   }
   
   handleSelectAccount() {
-    apiManager.glewlwydRequest("/auth/", "POST", {username: this.state.currentUser.username})
-    .then(() => {
-      messageDispatcher.sendMessage('App', {type: 'SelectAccountComplete'});
-    })
-    .fail(() => {
-      messageDispatcher.sendMessage('Notification', {type: "danger", message: i18next.t("login.error-login")});
-    });
+    messageDispatcher.sendMessage('App', {type: 'SelectAccountComplete'});
   }
   
   handleToggleGrantScope(user) {
-    this.setState({currentUser: user});
+    this.setState({currentUser: user}, () =>
+      apiManager.glewlwydRequest("/auth/", "POST", {username: this.state.currentUser.username})
+      .then(() => {
+        messageDispatcher.sendMessage('Notification', {type: "success", message: `${i18next.t("login.success-login")} (${this.state.currentUser.username})`});
```

Comment:
You don't need to use backquotes to surround the message, instead, add a new i18n entry:

```javascript
messageDispatcher.sendMessage('Notification', {type: "success", message: i18next.t("login.select-login", {username: this.state.currentUser.username})});
```

And the new i18n entry in the json locale files should be like:

```javascript
{
  "login": {
   // insert this in the "login block"
    "select-login": "User {{username}} selected"
  }
}
```